airbyte-ci/connectors/auto_merge/src/auto_merge/consts.py

```diff
@@ -4,6 +4,7 @@
 
 AIRBYTE_REPO = "airbytehq/airbyte"
 AUTO_MERGE_LABEL = "auto-merge"
+AUTO_MERGE_PROMOTED_RC_LABEL = "auto-merge/promoted-rc"
```

Comment:
I woud suggest finding a less specific name and defining a mapping of labels to validator groups. This would allow to easily expand your implementation to other use cases.
E.g : A PR has the `auto-merge` label AND `promoted-rc` -> use a specific set of validators. We could then add different combination with only a single change to make to a mapping.

Reply:
I've updated this by adding a `get_pr_validators` function and a label -> validator mapping dictionary. Take a look and let me know what you think.